### PR TITLE
Add Large Language Model provider

### PR DESCRIPTION
**Pull request by panilya**

### CODE REVIEW

File: src/main/resources/en/largelanguagemodel.yml
Comment (kingthorin) [inline src/main/resources/en/largelanguagemodel.yml:1]: No Grok? Is it based on one of these or are these only open models er something like that

**Review by bodiam — APPROVED** [review]

File: src/main/resources/en/largelanguagemodel.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - "Mistral Large"
+      - "Codestral"
+      - "Gemini 1.0 Pro"
+      - "Gemini 1.0 Ultra"
+      - "Gemini 1.5 Pro"
+      - "Gemini 1.5 Flash"
+      - "Gemma 7B"
+      - "LLaMA"
+      - "Llama 2"
+      - "Code Llama"
```
Comment (bodiam) [inline src/main/resources/en/largelanguagemodel.yml:42]: maybe add Grok-1 and OLMo?

File: src/main/resources/en/largelanguagemodel.yml
Comment (panilya) [reply src/main/resources/en/largelanguagemodel.yml:1]: I listed only popular models. Some are open-source models, and some aren't (like Llama by Meta vs GPT by <del>Closed</del>OpenAI).

You can check this website to see the ratings of LLM models: https://huggingface.co/spaces/lmsys/chatbot-arena-leaderboard. The leaderboard is based on people's ratings.

It's not based on anything. They say it was trained from scratch, uses a Mixture of expert techniques (like Mixtral 8x7B) etc.

File: src/main/resources/en/largelanguagemodel.yml (hunk 1 above)
Comment (panilya) [reply src/main/resources/en/largelanguagemodel.yml:42]: Added Grok for Grok's fans 😄

**Pull request merged**